go/teams/list.go

```diff
@@ -105,93 +226,41 @@ func List(ctx context.Context, g *libkb.GlobalContext, arg keybase1.TeamListArg)
 			if err != nil {
+				g.Log.CDebugf(subctx, "| Error in getTeamForMember: %v", err)
 				return err
 			}
 
 			type AnnotatedTeamInviteMap map[keybase1.TeamInviteID]keybase1.AnnotatedTeamInvite
-
-			loadTeamForAMI := func(force bool) (anMemberInfo *keybase1.AnnotatedMemberInfo, anInvites AnnotatedTeamInviteMap,
-				retry bool, err error) {
-
-				g.Log.CDebugf(subctx, "TeamList loading(%v, force=%v)", memberInfo.TeamID, force)
-
-				team, err := Load(subctx, g, keybase1.LoadTeamArg{
-					ID:          memberInfo.TeamID,
-					NeedAdmin:   isAdminSaysServer,
-					ForceRepoll: force,
-				})
-				if err != nil {
-					return nil, nil, false, err
-				}
-
-				role := keybase1.TeamRole_NONE
-				if memberInfo.Role != keybase1.TeamRole_NONE {
-					memberUV, err := team.chain().GetLatestUVWithUID(memberUID)
-					if err != nil {
-						// retryable because the role of this member might have changed since cache hit
-						return nil, nil, true, fmt.Errorf("did not find %v (for role %v)", username, memberInfo.Role)
-					}
-					role, err = team.chain().GetUserRole(memberUV)
-					if err != nil {
-						return nil, nil, false, err
-					}
-				}
-				if role != memberInfo.Role {
-					return nil, nil, true, fmt.Errorf("got unexpected role: %v", role.String())
-				}
-
-				anMemberInfo = &keybase1.AnnotatedMemberInfo{
-					TeamID:         team.ID,
-					FqName:         team.Name().String(),
-					UserID:         memberUID,
-					Role:           role,
-					IsImplicitTeam: team.IsImplicit(),
-					Implicit:       memberInfo.Implicit, // This part is still server trust
-					Username:       username.String(),
-					FullName:       fullName,
-				}
-
-				anInvites = make(AnnotatedTeamInviteMap)
-				if isAdminSaysServer {
-					anInvites, err = AnnotateInvites(subctx, g, team.chain().inner.ActiveInvites, team.Name().String())
-					if err != nil {
-						return nil, nil, false, err
-					}
-				}
-
-				return anMemberInfo, anInvites, false, nil
-			}
-
 			var anMemberInfo *keybase1.AnnotatedMemberInfo
 			var anInvites AnnotatedTeamInviteMap
 
-			// Load the team in order to use local data instead of server-trust for fields.
-			// Try once without ForceRepoll and then once with. In case the cache is stale.
-			// This could be simpler and faster if the server returned the latest seqno as a hint.
-			anMemberInfo, anInvites, retry, err := loadTeamForAMI(false)
-			if err != nil {
-				if retry {
-					anMemberInfo, anInvites, retry, err = loadTeamForAMI(true)
-				}
+			anMemberInfo = &keybase1.AnnotatedMemberInfo{
+				TeamID:         team.ID,
+				FqName:         team.Name().String(),
+				UserID:         memberInfo.UserID,
+				Role:           memberInfo.Role, // memberInfo.Role has been verified during getTeamForMember
+				IsImplicitTeam: team.IsImplicit(),
+				Implicit:       memberInfo.Implicit, // This part is still server trust
+				Username:       username.String(),
+				FullName:       fullName,
```

Comment:
Mention in a comment here that `Username` and `FullName` (for other users) are left blank and filled by `fillUsernames`. Is that right?